browser/src/control/Control.Shortcuts.js

```diff
@@ -0,0 +1,75 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Shortcuts for the classic toolbar & the Notebookbar.
+ * This is a duplicate of the shortcuts defined in `MenubarShortcuts` in Control.Menubar.ts.
+ * Once Control.TopToolbar.js and Control.Notebookbar*.js are converted to TypeScript, we can remove this duplication.
+ */
+
+/* global app _ L */
+
+L.Control.Shortcuts = {
```

Comment:
can we convert it to a ECMAScript class?

`L.Control.something` is a Leaflet-style convention.
This will be used in UI which should be all migrated to our "JSDialogs" so we can put it in `JSDialog` "namespace" I think.

```
class ShortcutsUtil {...}

JSDialog.ShortcutsUtil = new ShortcutsUtil();

```

similar to https://github.com/CollaboraOnline/online/blob/master/browser/src/control/jsdialog/Util.MessageRouter.ts